substrafl_examples/get_started/run_diabetes_substrafl.py

```diff
@@ -0,0 +1,573 @@
+    )
+    for i, org_id in enumerate(DATA_PROVIDER_ORGS_ID)
+}
+
+# %%
+# The data has now been added as an asset through the data samples.
+#
+# SubstraFL provides different type of Nodes to ingest these data, a Node being an object that will create an link
+# the different tasks with each other.
+#
+# An :ref:`Aggregation node<substrafl_doc/api/nodes:AggregationNode>` is attached to an organization (aka a Client)
+# and will be a node  where we can compute function that does not need data samples as input. We will use the
+# :ref:`Aggregation node<substrafl_doc/api/nodes:AggregationNode>` object to compute the aggregated analytics.
+#
+# A :ref:`Train data node<substrafl_doc/api/nodes:TrainDataNode>` is a Node attached to an organization (aka a Client)
+# and will have access to the data samples. These data samples must be instantiated with the right permissions to
+# be processed by the given Client.
+#
+# A third type of node exists in SubstraFL, called :ref:`Test data node<substrafl_doc/api/nodes:TestDataNode>`.
+# We will not need it in the current example. See the `MNIST Example
+# <https://docs.substra.org/en/stable/substrafl_doc/examples/get_started/run_mnist_torch.html#sphx-glr-substrafl-doc-examples-get-started-run-mnist-torch-py>`__
+# to learn how to use this type of Nodes.
+
+from substrafl.nodes import TrainDataNode
+from substrafl.nodes import AggregationNode
+
+
+aggregation_node = AggregationNode(ANALYTICS_PROVIDER_ORG_ID)
+
+# Create the Train Data Nodes (or training tasks) and save them in a list
+train_data_nodes = [
+    TrainDataNode(
+        organization_id=org_id,
+        data_manager_key=dataset_keys[org_id],
+        data_sample_keys=[datasample_keys[org_id]],
+    )
+    for org_id in DATA_PROVIDER_ORGS_ID
+]
+
+
+# %%
+# The ComputePlanBuilder class
+# ============================
+#
+# This example aims at explaining how to use the :ref:`substrafl_doc/api/compute_plan_builder:Compute Plan Builder`
+# class, and how to use the full power of the flexibility it provides.
+#
+# The :ref:`substrafl_doc/api/compute_plan_builder:Compute Plan Builder` is an abstract class that asks the user to
+# implement only three methods:
+#
+# - ``build_compute_plan(...)``
+# - ``load_local_state(...)``
+# - ``save_local_state(...)``
+#
+# The ``build_compute_plan`` method is essential to create the graph of the compute plan that will be executed on
+# Substra. Using the different nodes we created, we will update there states by applying user defined methods,
+# called ``RemoteMethod`` or ``RemoteDataMethod``, created using simply decorators, such as ``@remote`` or ``@remote_data``.
+#
+# These methods are pass as argument to the node using there ``update_state`` method.
+#
+# The ``update_state`` method outputs the new state of the node, that can be passed as an argument to a following one.
+# This succession of ``next_state`` pass to new ``node.update_state`` is how Substra create the graph of the
+# ``ComputePlan``.
```

Comment:
Let's be consistent with either ``ComputePlan`` or .ComputePlan.
And if we do use ``ComputePlan`` then I would use ``Node`` as well

Reply:
I'm not sure to understand what you meant :/